content/en/tutorials/vpc_peering.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "AWS VPC Peering"
```

Comment:
General question, this tutorial is suggested to go to AWS console and manually configuring VPC peering.
- Is there another way to configure VPC peering that is IaC?
- Is that something we could solve with opta without requiring going to AWS console?

Reply:
Is there another way to configure VPC peering that is IaC? -- Yes, but it is tricky as in terraform the peering connection needs to be created and accepted, which can't be done if it's cross-region or account. Maybe with something like the "delegated" flag it can be done.

Reply:
Is that something we could solve with opta without requiring going to AWS console? -- see above

Reply:
is that the accepted part that is tricky since it requires a UI approval?

This could be a good lead if Opta can do something better that terraform on this topic, may be we can keep an open ear on this topic.